components/search/SearchBar.vue

```diff
@@ -77,6 +79,10 @@ export default class SearchBar extends mixins(
     return this.inputFocused ? '' : this.$t('general.searchPlaceholder')
   }
 
+  get maxHeight() {
+    return this.istouch ? '100vh' : '600px'
+  }
+
```

Comment:
Could we try to do this in CSS way? For example, we could remove the `max-height` config and add some CSS for it. So that we could remove the new mixin `ScreenDimesionsMixin`.

Reply:
The beufy autocomple component needs it as a prop 
Otherwise default value is 200px

https://buefy.org/documentation/autocomplete/#api-view

Can try and override it with css if you feel that would be a better solution

Reply:
Yes, try to override it with CSS. It could avoid introducing extra mixin, only for the tiny style change.

Reply:
fixed.
indeed simple and easy. no need for mixin
thank you :smile: